File: q26.cpp
function removeDuplicates(nums: number[]): number {
    let slow = 1;
    for(let i = 1;i<nums.length;i++){
        if(nums[i]!= nums[i-1]){
            nums[slow]= nums[i];
            slow++;
        }
    }
    return slow;

};

File: q26.ts
function removeDuplicates(nums: number[]): number {
    let resultLength = 1;
    for (let i = 1; i < nums.length; i++) {
        if (nums[i] != nums[i - 1]) {
            nums[resultLength] = nums[i];
            resultLength++;
        }
    }
    return resultLength;
};
